documentation/src/main/asciidoc/topics/proc_configuring_memory_size_bytes.adoc

```diff
@@ -0,0 +1,61 @@
+[id='configure_eviction_size-{context}']
+= Configuring the Maximum Amount of Memory for {brandname} Caches
+Limit the size of the data container for cache entries to a maximum amount of
+memory. Performing eviction based on the maximum amount of memory requires you
+to use a binary-based storage format for the data container.
+
+.Procedure
+
+. Configure your {brandname} cache to use a storage format that supports binary encoding.
++
+[NOTE]
+====
+Configuring the encoding for caches is optional. {brandname} uses the MediaType
+that is determined by user configured marshallers if you do not configure
+encoding for caches.
```

Comment:
Suggestion:

```
It is recommended to configure the encoding for caches.
If it is not configured, {brandname} uses the MediaType of the marshaller from the `serialization` 
configuration of the cache container. 
```

Reply:
@gustavonalle I'd like to expand this a little to understand why it's recommended. Could there be cases where the MediaType of the marshaller is not suitable for caches that require a binary storage format? So it's better to set the encoding for each cache to potentially avoid issues?

Reply:
> Could there be cases where the MediaType of the marshaller is not suitable for caches that require a binary storage format

All marshallers should produce `byte[]`, so there is not a problem

> So it's better to set the encoding for each cache to potentially avoid issues?

The issue is that the marshaller config is also optional, so it ends up picking whatever is the "default" internally in Infinispan. To make things worse, the "default" historically (up to 10) has been different for embedded and server caches, so an OFF_HEAP cache would store different formats if used by the client or server. Setting an encoding will guarantee a stable format for the cache until this marshaller mess is fully sorted out.

Reply:
this is great. it's actually made me think a little more about the global configuration and how we are inconsistent with that in docs. hopefully the changes look good.